content/guides/getting_started.adoc

```diff
@@ -35,16 +35,39 @@ brew upgrade clojure
 
 === Installation on Linux
 
-1. Ensure that the following dependencies are installed: bash, curl, rlwrap, and Java.
-2. Use the linux-install script to download and run the install, which will create /usr/local/bin/clj, /usr/local/bin/clojure, and /usr/local/lib/clojure:
+1. Ensure that the following dependencies are installed: `bash`, `curl`, `rlwrap`, and Java (Java JDK 11, 9 and 8 should be acceptable but you may want to [choose between different vendors](https://purelyfunctional.tv/guide/clojure-jvm/#which-jdk) ).
```

Comment:
Why is Java 10 missing from the list? Really, I'd prefer to say less if possible  (the downloads page has more info on Java compatibility and should be the better long-term source for this info). The external link here is good info but I wonder if we should address both of these directly, or perhaps do that in a footnote.

Reply:
You are right, I don't know why I left out Java 10. Actually, Java 9 and 10 are just "short time steps" between 8 and 11. I suspect that if there hadn't been marketing, one would have seen Java 9.0, 9.1 (instead of 10) and 9.2 (instead of 11). This wikipedia page has a clear overview: https://en.wikipedia.org/wiki/Java_version_history - maybe one should have a similar table right here.

Reply:
Clojure Contributor Agreement signed! :ok_hand: